torch/lib/c10d/reducer.cpp

```diff
@@ -1498,16 +1526,18 @@ void Reducer::set_parameter_stats() {
   }
 }
 
-void Reducer::set_bucket_stats() {
+std::string Reducer::get_bucket_stats() {
```

Comment:
Why is this a string?

Reply:
If the stats only mean "sizes", it will be better to rename this function as "get_bucket_sizes". Stats sound very vague.

Reply:
because some logger only support string, but you are right, let me change it back to vector<int> in struct level, and log it as string for some logger